httpx/access_test.go

```diff
@@ -14,20 +14,35 @@ import (
 func TestAccessConfig(t *testing.T) {
 	defer httpx.SetRequestor(httpx.DefaultRequestor)
 
-	access := httpx.NewAccessConfig(30*time.Second, []net.IP{net.ParseIP("127.0.0.1"), net.ParseIP("::1")})
+	access := httpx.NewAccessConfig(
+		30*time.Second,
+		[]net.IP{
+			net.ParseIP("127.0.0.1"),
+			net.ParseIP("::1"),
+		},
+		[]*net.IPNet{
+			{IP: net.IPv4(10, 0, 0, 0).To4(), Mask: net.CIDRMask(8, 32)},
```

Comment:
this is one of the IP ranges designated for private networks (see https://en.wikipedia.org/wiki/Private_network) so idea would be that mailroom can add all three to its config